code/game/objects/structures/crates_lockers/closets/coffin.dm

```diff
@@ -12,6 +12,18 @@
 	else
 		icon_state = icon_opened
 
+/obj/structure/closet/coffin/attackby(obj/item/W, mob/living/user)
+	if(istype(W, /obj/item/tool/crowbar))
+		var/obj/item/tool/crowbar/WT = W
+		new /obj/item/stack/sheet/wood(src.loc)
+		for(var/mob/M in viewers(src))
+			M.show_message(SPAN_NOTICE("\The [src] has been wrenched apart by [user] with [WT]."), 3, "You hear wood breaking.", 2)
+		qdel(src)
+		return
+	if(isrobot(user))
+		return
+	user.drop_inv_item_to_loc(W,loc)
```

Comment:
Give W, M, WT more descriptive variable names

Reply:
I've made the requested changes, the variables should be a bit better now

 `/obj/structure/closet/coffin/attackby(obj/item/Prybar, mob/living/user)
	if(istype(Prybar, /obj/item/tool/crowbar))
		var/obj/item/tool/crowbar/CrB = Prybar
		new /obj/item/stack/sheet/wood(src.loc)
		for(var/mob/Mob in viewers(src))
			Mob.show_message(SPAN_NOTICE("\The [src] has been wrenched apart by [user] with [CrB]."), 3, "You hear wood breaking.", 2)
		qdel(src)
		return
	if(isrobot(user))
		return
	user.drop_inv_item_to_loc(Prybar,loc)
`